packages/dapp/src/shared/OrderedInput.jsx

```diff
@@ -9,11 +9,46 @@ import {
   Tooltip,
   VStack,
 } from '@chakra-ui/react';
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 
 import { QuestionIcon } from '../icons/QuestionIcon';
 import { isValidLink } from '../utils/helpers';
 
+// const Requirement = (required) => {
+//   const [requirement, setRequirement] = useState('');
+
+//   useEffect(()=>{
+//     setRequirement(required)
+//   }),[required]
+
+//   return(
+//     <>
+//      {requirement === 'null' ?
+//     <Text
+//       fontStyle='italic'
+//       fontSize='10%'
+//       marginLeft='5px'
+//       paddingTop='5px'
+//     >
+//     {console.log(requirement)}
+//     {requirement === 'required' ? 'Required' : 'Optional'}
+//     </Text> :
+//     <Text
+//       fontStyle='italic'
+//       fontSize='10%'
+//       marginLeft='5px'
+//       paddingTop='5px'
+//       visibility='hidden'
+//     >
+//     Required
+//     </Text>
+//     }
+
+//     </>
+
+//   )
+// }
```

Comment:
You knew I was going to comment on this right? Does this need more work before ready? Maybe pull it to a separate branch.

Reply:
Lol how did I forget to remove that? 😅